### PR TITLE
Fix scrot version string / int comparison

### PR DESCRIPTION
**Pull request by hummusbird**

### CODE REVIEW

**Comment by hummusbird** [comment]
```
Traceback (most recent call last):
  File "/usr/bin/gscreenshot", line 33, in <module>
    sys.exit(load_entry_point('gscreenshot==3.1.2', 'gui_scripts', 'gscreenshot')())
  File "/usr/lib/python3.10/site-packages/gscreenshot/frontend/__init__.py", line 43, in delegate
    gscreenshot.frontend.gtk.main()
  File "/usr/lib/python3.10/site-packages/gscreenshot/frontend/gtk.py", line 925, in main
    application = Gscreenshot()
  File "/usr/lib/python3.10/site-packages/gscreenshot/__init__.py", line 68, in __init__
    self.screenshooter = screenshooter_factory.create()
  File "/usr/lib/python3.10/site-packages/gscreenshot/screenshooter/factory.py", line 47, in create
    if shooter.can_run():
  File "/usr/lib/python3.10/site-packages/gscreenshot/screenshooter/scrot.py", line 58, in can_run
    if float(scrot_version_num) >= 1:
```

File: src/gscreenshot/screenshooter/scrot.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def can_run() -> bool:
         """Whether scrot is available"""
         try:
             scrot_version_output = subprocess.check_output(['scrot', '--version'])
-            scrot_version_str = scrot_version_output.decode().strip().rsplit(' ', maxsplit=1)[-1]
+            scrot_version_str = scrot_version_output.decode().strip().rsplit('.', maxsplit=1)[-1]
             scrot_major_version = scrot_version_str.split('.')[0]
```
Comment (thenaterhood) [inline src/gscreenshot/screenshooter/scrot.py:56]: Hey there, thanks! With this change, this line (56) isn't really needed and we can just compare off of the scrot_version_str variable.

```
>>> foo = "scrot version 1.8.1".strip().rsplit('.', maxsplit=1)
>>> print(foo)
['scrot version 1.8', '1']
>>> foo = "scrot version 1.8.1".strip().rsplit('.', maxsplit=1)[-1]
>>> print(foo)
1
>>> print(foo.split('.')[0])
1
```

Comment (hummusbird) [reply src/gscreenshot/screenshooter/scrot.py]: Fixed!

**Comment by thenaterhood** [comment]
Looks good, thanks!

**Pull request merged**